src/test/librados/crimson_env.h

```diff
@@ -0,0 +1,12 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#pragma once
+
+#include <cstdlib>
+
+#define CHECK_ENV();                    			\
+  if (getenv("CRIMSON_COMPAT")) {  			     	\
+    GTEST_SKIP() << "Not supported, Skipping single test";	\
+  }
```

Comment:
Let's name this SKIP_IF_CRIMSON()

Let's also introduce a helper is_crimson_cluster() for...